src/desktop/components/main_layout/templates/blank.jade

```diff
@@ -14,7 +15,7 @@ html( data-useragent= userAgent lang="en")
     if assetPackage
       link( type='text/css', rel='stylesheet', href= asset('/assets/#{assetPackage}.css') )
 
-  body( class= bodyClass )
+  body( class= bodyClass )&attributes(attributes)
```

Comment:
I've never seen this syntax before?

Reply:
http://jade-lang.com/reference/attributes#and-attributes